test/components/tree-view.spec.tsx

```diff
@@ -562,5 +562,80 @@ describe('<TreeView />', () => {
                 expect(last.label).to.eql(treeData[0].children![2].label);
             });
         });
+
+        describe('TreeView methods', () => {
+            const firstChild = treeData[0].children![0];
+            const secondChild = treeData[0].children![1];
+
+            // trying to put the type of waitForDom that contained type 'Function' gives
+            // a lint error so i used any for now
+            async function expandPartsOfTree(select: any, waitForDom: any) {
```

Comment:
Consider instead of this to create a method called `renderTreeAndExpand(React.Element)`

Reply:
Will implement